postgrest.dict

```diff
@@ -212,16 +183,14 @@ uri
 url
 urls
 variadic
-Vercel
 verifier
 versioning
 Vondra
 Vue
-WAI
 webhooks
 websearch
 Websockets
 webuser
 wfts
 Zac
-ZeroMQ
+ITSHOULDFAIL
```

Comment:
This one is added on purpose to see CI fail right?

Reply:
Yes, I just force pushed without that entry.